### PR TITLE
#164325146 Update database name to a more descriptive name

### PR DESCRIPTION
**Pull request by AlvinMugambi**
**What does this PR do**
Updates the database name from drf to ireporter_db

**Tasks to be completed**
- Change database name in the env-example from drf to ireporter db
- Update database name in the readme to ireporter_db

**Relevant PT story**
[#164325146](https://www.pivotaltracker.com/story/show/164325146)

### CODE REVIEW

**Comment by katherine95** [comment]
Squash commits into one commit.

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (katherine95) [inline .gitignore]: This conflict needs to be solved.

Comment (katherine95) [reply .gitignore]: Resolve then push. There must have been a merge conflict.

Comment (meshack-mbuvi) [inline .gitignore]: Please, check that you have resolved conflicts very well. You can reach out to @katherine95 in case you are blocked

Comment (AlvinMugambi) [reply .gitignore]: Resolved

Comment (AlvinMugambi) [reply .gitignore]: Resolved

**Comment by AlvinMugambi** [comment]
> Squash commits into one commit.

Done.

**Pull request merged**